examples/compiled/rect_binned_heatmap.vg.json

```diff
@@ -16,10 +16,6 @@
             },
             "transform": [
                 {
-                    "type": "filter",
```

Comment:
It looks like moving the filter affects the bin for this example.   Not sure why.  Maybe the bin operator does not handle invalid filter well?